socorro/unittest/external/postgresql/unittestbase.py

```diff
@@ -19,11 +21,15 @@ class PostgreSQLTestCase(TestCase):
     required_config = Namespace()
     required_config.namespace('database')
     required_config.add_option(
+        name='database_url',
+        default='postgres://test:aPassword@localhost:5432/socorro_integration_test',
+        doc='Name of database to manage',
+    )
+    required_config.add_option(
```

Comment:
I was trying not to make the heroku-specific stuff be mandatory. Having the password embedded in the URL might be an issue for some users as well.